### ISSUE TITLE
Creating Task in existing project from template

### ISSUE THREAD

**Issue by AlexisMaizel** [ksalzke/templates-for-omnifocus#39]
Hi Kaitlin,
Thanks for the plugin; it is a great time saver. 
Would it be possible to adapt the plugin to programmatically allow the creation of Task(s) inside an already given project? 

If $FOLDER is undefined, one can select an existing project, and the template elements are added as tasks to the project. This is great, but would it be possible to define a  `$PROJECT=project_name` tag to specify the target project? That would save time. 

Side notes: 
- I second #28 as a feature request
- Unless I am mistaken, when prompted for a destination and selecting 'Show project', all projects are listed (including the completed ones).

Keep up to good work!

**Comment by ksalzke** [comment]
Yes, theoretically possible. Thanks for the suggestion - I've noted it, but will say that it will probably be some time before I get around to implementing it.

And I think you're right about your second side note - I've added a separate issue (#40).

Thanks!
